pkg/controller/datavolume/controller-base.go

```diff
@@ -694,6 +694,11 @@ func (r *ReconcilerBase) reconcileProgressUpdate(datavolume *cdiv1.DataVolume, p
 		return nil
 	}
 
+	// With upload we don't have report of progress and no need to requeue
+	if datavolume.Spec.Source != nil && datavolume.Spec.Source.Upload != nil {
```

Comment:
Should base controller really be doing upload specific stuff?  Can it be moved to upload controller?

Reply:
Or something added to import?